src/components/layouts/CommitBar.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { colors } from '../../constants/colors';
+
+const StyledWrapper = styled.div`
+  min-width: 250px;
+  background-color: ${colors.background.GREY_3};
+  color: ${colors.font.WHITE};
+`;
+
+export default function CommitBar({ children }) {
+  return (
+    <>
+      <StyledWrapper>{children || ''}</StyledWrapper>
+    </>
```

Comment:
```suggestion
      <StyledWrapper>{children || ''}</StyledWrapper>
```

여기도  fragment 필요없을 것 같습니다